lib/gitgitgadget.ts

```diff
@@ -173,10 +173,15 @@ export class GitGitGadget {
     public async preview(pr: IPullRequestInfo, userInfo: IGitHubUser):
         Promise<IPatchSeriesMetadata | undefined> {
 
+        if (!userInfo.email) {
```

Comment:
CIHelper will have already thrown an error if there is no public email for the `/preview` command.

Reply:
That may be so, but I don't think that any static analysis can take that into account because `preview()` might be called from elsewhere, right?

Reply:
Yes, `preview()` could be called from elsewhere.  Should the check on email be removed from CIHelper, delegating the check to `GitGitGadget::preview`?  Removes a double check and allows email to be filled in by `CIHelper::checkCommits`?

This could be done as a new PR.

Reply:
Hmm. I think it is fine to have both. The CIHelper can give a better error message, and the `preview()` method needs to check as a last line of defense.